monitoring/using-rfhe.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="using-rfhe"]
+= Monitoring Redfish hardware events
+include::_attributes/common-attributes.adoc[]
+:context: using-rfhe
+
+toc::[]
+
+:FeatureName: Redfish hardware events
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="about-using-redfish-hardware-events"]
+== About Redfish hardware events
+
+{product-title} enables your monitoring services to consume cloud-native events from Redfish hardware event-capable equipment. The events are published on a node and transmitted on an advanced message queue to your subscribing applications.
+
+Redfish hardware events provide the basis for a standard network monitoring services that is cloud friendly and secure. The standard is developed under the guidance of the DMTF (formerly referred to as the Distributed Management Task Force). Redfish provides an industry-standard protocol with a RESTful interface. The protocol can be used for the management of distributed, converged or software-defined resources and infrastructure.
+
+[NOTE]
+====
+Redfish hardware events work only with Redfish-capable devices on OpenShift single-node clusters provisioned on baremetal infrastructure.
+====
+
+The hardware-related events published through Redfish to monitoring applications may include:
+
+* breaches of temperature limits
+* server status
+* router status
+* fan status
+
+Application developers can use the Redfish hardware events API to get log updates on things that have or could go wrong in a distributed system.
```

Comment:
Change "... on things that have or could go wrong..." to "...errors that have or could occur.."

Reply:
Done.